pom.xml

```diff
@@ -201,7 +201,7 @@
     </dependency>
     <dependency>
       <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-data-pipeline</artifactId>
+      <artifactId>cdap-data-pipeline3_2.12</artifactId>
```

Comment:
Are we not supporting version 2.x?

Reply:
I confirmed with @fernst that we no longer support 2.x.

Reply:
We removed support for Scala 2.11